### ISSUE TITLE
Inconsistent / Misleading terms when Entering Product & Variant Details  

### ISSUE THREAD

**Issue by emilyjeanrogers** [openfoodfoundation/openfoodnetwork#4788]
## Description

When entering product information in Admin, field labels are not descriptive and in some cases are quite misleading. They do not accurately represent the content required and while entering data there is no way to confirm what content should go where, or understand how it will be displayed. The only way to know if you’ve entered content correctly is to set up an order cycle and view a live shopfront. 

Examples:
1. Weight is sometimes referred to as **Unit** and sometimes as **Weight**. The manual override field for Weight is labelled ***Display As.**

2. The override field to manually change how Weight appears is labelled **Display as**. On the same screen, Variant Name is labelled **Display Name** . It is not clear that one field is for entering a weight and the other is for entering a name.

3. Product Variant is never referred to as **Variant**. It is either labelled **Name** or **Display Name**. The Product Name is also referred to as **Name**, and is not distinguished from Variant Name

## Expected Behaviour

The fields would be labelled intuitively. Proposed label names:

1.  Labels referring to Product:  PRODUCT NAME
2.  Labels referring to Variant: either VARIANT NAME or PRODUCT VARIANT NAME
3.  Labels referring to Weight: WEIGHT
3.  Labels referring to how WEIGHT appears in shopfront: DISPLAY WEIGHT 

This content appears in the Shopfront as follows:

(Each piece of content has been noted in red using the above intuitive names)

![Display Name Confusion 3](https://user-images.githubusercontent.com/47620442/74390318-0289e680-4e55-11ea-970b-f9985dcb01ee.png)

This content appears in a single line for each product on the checkout screen 

![Display Name Confusion 4](https://user-images.githubusercontent.com/47620442/74390486-8cd24a80-4e55-11ea-8ba9-1100519584c5.png)

## Actual Behaviour

**Bulk Edit Products Overview Screen**

- Product Name and Variant Name appear in the same column, which is labelled NAME.  
- Weight appears in a column labelled UNIT 
- A manual override field changing how Weight is displayed is labelled DISPLAY AS.

(Proposed / intuitive labels marked in red)

![Display Name Confusion 2](https://user-images.githubusercontent.com/47620442/74391171-85ac3c00-4e57-11ea-9e1a-17d69d56d826.png)

**Individual Product Edit Screen**

When entering data about an individual product:

- Product Name (previously referred to as NAME) appears as the page heading
- The Product Variant Name is referred to as DISPLAY NAME
- Manual override for the Weight of the Product is referred to as DISPLAY AS 
- The actual weight (beneath this & previously referred to as UNIT) now labelled WEIGHT (KG) 

None of these fields have descriptors, and it is not clear how to differentiate between Display Name, Display As and Weight (KG). Previously used labels Name and Unit are not referenced on this screen.

(Proposed / intuitive labels marked in red)

![Display Name Confusion 1](https://user-images.githubusercontent.com/47620442/74391276-d6239980-4e57-11ea-8a62-0c543398a05a.png)

## Steps to Reproduce

1. Log in as an Enterprise with products added
2. Edit Products in Bulk Edit Screen
3. Edit Products in Individual Product Edit Screen

## Workaround
It is possible to enter all product information, however the only way to know if you’ve entered content correctly is to set up an order cycle and view a live shopfront. 

## Severity
S4

## Possible Fix

Change labels in Bulk Edit Products and Individual Edit Products data entry screens.

**Comment by Matt-Yorkley** [comment]
The `display_as` field doesn't map exactly to "display weight". Aside from that, I agree the Product and Variant field names need some better explanations in the UI. Maybe some tooltips would be the best option, and they could be reused throughout the app.

On a tech tangent; it would be nice from a multilingual perspective if we could conditionally show the new tooltips only if the relevant translation key exists, so they only appear in the UI for each instance once they've been translated.

**Comment by luisramos0** [comment]
yes Matt, unit is never weight, unit is always "weight, volume or items" so it really is "display weight, display volume" or "display items".
The "product variant unit name" can be weight volume or items and will change the "display as" field. And we can change the labels based on the "product variant unit name".
maybe we can improve by just changing "display as" to "display unit as".
Otherwise, we can:
if "product variant name" = weight we can show "display weight"
if "product variant name" = volume we can show "display volume"
if "product variant name" = items we can show "display items (unit)" or "display items name"

**Comment by luisramos0** [comment]
if just changing the labels, this can be a papercut :+1:

**Comment by Erioldoesdesign** [comment]
Yeah agreed that sometimes the 'unit' and 'display as' is an item and therefore defined as 'bunches' of carrots 'head' of broccoli 'slice' of pastry etc.

When you have a field that can be that generalised the label also needs to be generalised otherwise they won't make sense and be odd wording. And in a product list you can have multiple kinds of unit if you're a producer or hub selling say, potatoes by the kg, carrots by the bunch and cakes by the cake. So unit or maybe 'size' are the generalised terms that can cover this kind of list.

I think changing the labels here should be done with some testing with users and close monitoring. 

I don't think that it's as simple as 'changing the labels' so I would say this papercut need design investigation that would likely take a while.

**Comment by luisramos0** [comment]
I am ok with that decision :+1: 
I wonder what is the process here, move this issue to the design GH repo so we build a backlog there of things like this?

**Comment by Erioldoesdesign** [comment]
I think that's a good idea @luisramos0 

Pinging @RachL @jaycmb @kirstenalarsen and @lin-d-hop to give a 'thumbs up' if this can be transferred into the 'Inception pipe' because it needs design but not 'papercut length design' as in, design will take longer than a few hours, therefore, it needs to be a design task in the design part of the inception pipe.

**Comment by RachL** [comment]
@Erioldoesdesign I'm not sure it is something we should focus on soon. In any case the work that OFN global is doing on this can only end up with some recommendation in English. Instances (even in English) will in any case override these labels with their translation files.

**Comment by Erioldoesdesign** [comment]
Agreed it's not something to focus on, but was suggesting it go in an 'icebox' or 'backlog' for design @RachL

**Comment by kirstenalarsen** [comment]
@Erioldoesdesign I am putting this in the now-existing papercuts for design, but am NOT suggesting design time should be spent on it now. I would be inclined to say that the likely time required for design / planning of this would mean it is not a papercut at all and should be removed from the board - your thoughts?

**Comment by Erioldoesdesign** [comment]
@kirstenalarsen seems like from all the discussion it's got a lot more to it. Agreed re. remove from board.

Glancing at the content it might be a candidate for future phases of UI uplift to attempt to solve. At the very least UI uplift changes some of these inputs so this issue would need to evolve once that piece is done :)
